modules/ROOT/pages/index.adoc

```diff
@@ -43,20 +44,19 @@ You can either download one of our example data set or model and import your own
 [.column]
 === 3. Build your application
 
-You can use our xref:graph-platform.adoc[suite of Neo4j products] to share your data or build out your own application in the programming language of your preference.
+Use the xref:graph-platform.adoc[Neo4j tools] to access your data or develop an application your favorite programming language.
```

Comment:
Maybe `use neo4j tools` rather than `use the neo4j tools`?

Then the last line needs a "with" or "in": `develop an application with your favorite programming language.`